blues/application/tasks.py

```diff
@@ -11,6 +11,7 @@
 from .providers import get_providers
 
 from .. import git
+from .. import slack
```

Comment:
LGTM, PEP-8 allows both styles for `from _ import _`  -- https://www.python.org/dev/peps/pep-0008/#imports